odc/geo/ui.py

```diff
@@ -84,7 +84,7 @@ def svg_base_map(
     sz=360,
     map_svg_path=None,
     target=None,
-):
```

Comment:
Since we are making this typed function, lets add input side types also. Same applies for function below `make_svg`

Reply:
Fixed.